kong/clustering/init.lua

```diff
@@ -70,7 +70,11 @@ function _M:init_dp_worker(plugins_list)
       return
     end
 
-    local config_proto, msg = check_protocol_support(self.conf, self.cert, self.cert_key)
+    local config_proto, msg
+    if not kong.configuration.force_no_wrpc then
```

Comment:
`force_no_wrpc` is a little strange, could we rename it to `legacy_hybrid_protocol` or else?

Reply:
Maybe `hybrid_force_legacy_protocol`?

Reply:
or `hybrid_legacy_protocol` for consistency with `legacy_worker_event`. (hybrid is the scope.)